src/wh_server_crypto.c

```diff
@@ -81,6 +139,88 @@ int wh_Server_HandleCryptoRequest(whServerContext* server,
     case WC_ALGO_TYPE_PK:
         switch (packet->pkAnyReq.type)
         {
+#ifndef NO_RSA
+        case WC_PK_TYPE_RSA_KEYGEN:
```

Comment:
Need to add some per-action packet size validations.  I'll add a backlog entry.  Ok for now.